FreshRSS/FreshRSS
# Improved: error page

Before:
![grafik](https://user-images.githubusercontent.com/1645099/181269276-1be3dea4-8065-4a62-8e27-4518499bfa14.png)

After:
![grafik](https://user-images.githubusercontent.com/1645099/181269348-0f5565dd-718a-4bde-bfcb-bf611acd5e62.png)



Changes proposed in this pull request:

- the error message is now a real HTML document (add: `<!DOCTYPE><html><header><title><body>`.... `</body></html>`
- error message is now a HTTP 500 error
- error message has now a CSP header
- copied the technical information to our documentation, cut out the text from the error message and added a link to the documentation
- the log file name is now defined as constant

How to test the feature manually:

1. go to the logs page
2. manipulate the page attribute (f.e. `page=abc`)
3. check the error message
4. see the 500 error in the browser's development tool

Pull request checklist:
